docs/setup.md

```diff
@@ -99,6 +99,8 @@ the *base (root) environment*, click the arrow next to it and click `Open termin
 Type following commands line by line + ENTER-button (make sure you have an internet connection):
 
 ```
+conda install -n base conda-libmamba-solver
+conda config --set solver libmamba
```

Comment:
This is probably not needed anymore, it should now be the default (https://conda.org/blog/2023-11-06-conda-23-10-0-release/)

Reply:
Could we keep it one more edition, so people who would not update an existing conda are also using libmamba?

Reply:
Sounds good